src/main/java/com/github/markusbernhardt/seleniumlibrary/keywords/Element.java

```diff
@@ -25,1075 +26,1093 @@
 @RobotKeywords
 public class Element extends RunOnFailureKeywordsAdapter {
 
-	/**
```

Comment:
:-( That's a little unexpected that now nearly all the lines were modified..

Reply:
ah, now I see
yes, I converted tabs to spaces, thinking my line was the only line with this problem.
Since this is just about the only change in this commit, it shouldn't be too bad though